src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java

```diff
@@ -32,18 +35,22 @@ public void updateIntegrationFlows() {
   }
 
   private void createFlows(Configuration configuration) {
+    log.debug("createFlows:: Creating flows for configuration: {}", configuration.getId());
     registerRetrievalsPoller(configuration);
     registerRetrievalQueueRecordFlow(configuration);
   }
 
   protected void removeExistingFlows() {
+    log.debug("removeExistingFlows:: Trying to remove if there are any Existing Flows");
     integrationFlowContext.getRegistry().keySet().forEach(key -> {
       integrationFlowContext.getRegistrationById(key).stop();
       integrationFlowContext.remove(key);
     });
   }
 
   public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalsPoller(Configuration configuration) {
+    log.debug("registerRetrievalsPoller:: Registering retrievals poller for configuration: {}", configuration.getId());
+    log.info("registerRetrievalsPoller:: Registered retrievals poller for configuration: {}", configuration.getId());
```

Comment:
Please remove info log and keep single logger statement as method itself is having single line.

Reply:
removed info log.